src/JWT/Algorithms/RS256Algorithm.cs

```diff
@@ -91,4 +91,4 @@ private static RSACryptoServiceProvider GetPublicKey(X509Certificate2 cert)
             return (RSACryptoServiceProvider)alg;
         }
     }
-}
\ No newline at end of file
+}

```

Comment:
Please also bump the minor version in JWT.csproj so I could publish this change to NuGet. Otherwise it'll go live only next time (and who knows when it'll happen?)

Reply:
done!